guides/doc-Release_Notes/topics/foreman-3.3.1.adoc

```diff
@@ -0,0 +1,79 @@
+= Foreman 3.3.1
+
+A full list of changes is available via https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1600[Redmine]
+
+== Foreman
+
+=== Templates
+
+* Restart yggdrasil from kptm if it is already running - https://projects.theforeman.org/issues/35472[#35472]
+* ks= kernel parameter in Kickstart default iPXE causes RHEL9 Anaconda failure to start - https://projects.theforeman.org/issues/34975[#34975]
+* Allow setting up rex pull during provisioning and registration - https://projects.theforeman.org/issues/34969[#34969]
+* Ubuntu Autoinstall PXELinux template does not reference Smart Proxy correctly - https://projects.theforeman.org/issues/34941[#34941]
+
+=== Compute resources - VMware
+
+* hammer cannot use the cluster name or id as valid input when clusters are residing inside folders and fails with error fog not found - https://projects.theforeman.org/issues/35438[#35438]
+
+=== Tests
+
+* Update show_me_the_cookies to 6.x - https://projects.theforeman.org/issues/35371[#35371]
+
+=== Settings
+
+* Ignore calico interfaces from Foreman - https://projects.theforeman.org/issues/35289[#35289]
+* Fix the SettingValueException error message - https://projects.theforeman.org/issues/34921[#34921]
+
+=== Host creation
+
+* Bulk rebuild hosts can not reset the build token - https://projects.theforeman.org/issues/35203[#35203]
+
+=== Unattended installations
+
+* Add installation medium for Stream 9 - https://projects.theforeman.org/issues/35151[#35151]
+
+=== Audit Log
+
+* Audit page shows "auditable id / Host2" for "Host1" but Host2 does not exist or deleted from the all hosts - https://projects.theforeman.org/issues/35132[#35132]
+
+=== Inventory
+
+* Hide reports "Origin" column when table is already filtered by origin - https://projects.theforeman.org/issues/35126[#35126]
+* Hosts power status ping is limited to three secondss power status ping is limited to three seconds - https://projects.theforeman.org/issues/35083[#35083]
+
+=== Host registration
+
+* Global Registration, if you use another LANG like pt_BR.UTF-8, it will broken the UUID - https://projects.theforeman.org/issues/35051[#35051]
+
+=== Smart Proxy
+
+* Smart Proxy upgrade/install fails due to proxy configuration in  HTTP(S) proxy' in settingsHTTP(S) proxy in settings in settings - https://projects.theforeman.org/issues/34996[#34996]
```

Comment:
wat :)

Reply:
Oh, there's a very weird issue with quotes: here I try to escape it to make it asciidoc safe, but it has "some issues":
https://github.com/theforeman/foreman-documentation/blob/3357580287bebfa749158d3c499ebc4802d5258c/guides/doc-Release_Notes/redmine_release_notes#L64
However, I never really understood why.

Reply:
Ah, needs more backslashes: https://stackoverflow.com/questions/2180322/ruby-gsub-doesnt-escape-single-quotes